ext/standard/tests/hrtime/hrtime.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Test hrtime() returns nanoseconds
+--FILE--
+<?php
+
+$time = hrtime();
+usleep(500000);
```

Comment:
How might sleep freeze some systems? Adding random computations doesn't guarantee it waits at least the time we test against.